### ISSUE TITLE
[SF][Badge]:   Fails to meet color contrast ratio with respect to background color when color scheme is "8"

### ISSUE THREAD

**Issue by sfsf-xweb-sh** [SAP/ui5-webcomponents#3776]
**Describe the bug**

 Badge Fails to meet color contrast ratio with respect to background color when color scheme is "8"
Current ratio is 2.5:1
Foreground Color : #80A854
Background Color : #F3F4EF

**To Reproduce**
1. open test environment: https://qacand.hcm.ondemand.com/sf/ompassignment?mock#/assignment-detail/ao/2

**Expected behavior**
Ensure the foreground color and background color of each element have a contrast ratio of at least 4.5:1

**Screenshots**
![image](https://user-images.githubusercontent.com/78520316/131594730-1b70436b-d262-4907-9dbb-ad054455bfb5.png)

**UI5 Web Components for React Information**
`@ui5/webcomponents` version:  o.31.14
`@ui5/webcomponents-react` version: 0.131.13

Browser: Chrome

**Additional context**
Please check it and fix it as soon as possible. Thanks a lot！

**Comment by Shtilianova** [comment]
Hello @SAP/ui5-webcomponents-topic-rd colleagues,
This is a contrast ratio issue for Badge. You could check it also here [https://sap.github.io/ui5-webcomponents/master/playground/components/Badge/](https://sap.github.io/ui5-webcomponents/master/playground/components/Badge/) since the reporter system is not accessible.
Regards, Diana

**Comment by sfsf-xweb-sh** [comment]
Hi @gmkv 
Please add the fix code to next new release 0.31.x.
Thanks!

**Comment by fifoosid** [comment]
Both changes(#3870 & #3814) are downported in the 0.31 branch
